spec/controllers/manifest_controller_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe ManifestsController, type: :controller do
+  describe 'GET #show' do
+    ['production', 'sandbox', 'demo', 'qa', 'development'].each do |env|
+      it "for #{env}" do
+        ENV['SIMPLE_SERVER_ENV'] = env
```

Comment:
Its better to stub it like this since it won't affect the other tests. 
```suggestion
        expect(ENV).to receive([]).with('SIMPLE_SERVER_ENV').and_return(env)
```